modules/zivid/captureassistant.py

```diff
@@ -0,0 +1,82 @@
+"""Contains capture assistant functions and classes."""
+import _zivid
+import zivid._settings_converter as _settings_converter
+from zivid._make_enum_wrapper import _make_enum_wrapper
+
+
+AmbientLightFrequency = _make_enum_wrapper(
```

Comment:
If this function call fails we will get exceptions when calling `import zivid`

Reply:
Is this different to other parts of our code which may fail?
Any suggestions to make it more robust?

Reply:
Can we not call this where we use this?

Reply:
It's needed when the captureassistant module is imported, so the autocomplete will work. As far as I can see, all the submodules are imported when zivid is imported.